tests/cbmc/include/cbmc_proof/cbmc_utils.h

```diff
@@ -25,7 +25,7 @@
 #define IMPLIES(a, b) (!(a) || (b))
 
 struct store_byte_from_buffer {
-    size_t  index;
+    size_t  position;
```

Comment:
I think fields in a structure are fine

Reply:
not sure you can distinguish between structs and arguments in the grep script, though

Reply:
Yeah probably correct :(